apps/package.json

```diff
@@ -289,6 +290,7 @@
     "react-motion": "^0.5.2",
     "react-onclickoutside": "~5.11.1",
     "react-paginate": "^6.3.0",
+    "react-papaparse": "^4.4.0",
```

Comment:
How do we feel about adding new packages? fwiw we used this same package for AI Lab 
- [ ] link to ml-playground code

Reply:
I think we are generally fine with it? That one looks widely used and has been updated in the last year, which both seem like good signs.